lib/routes/account.js

```diff
@@ -1046,31 +1043,15 @@ module.exports = function (
 
         function createResponse() {
 
-          var doSigninConfirmation = requestHelper.shouldEnableSigninConfirmation(sessionToken.uid.toString('hex'), config)
```

Comment:
I think it might be OK to just return these extra fields regardless of whether the user has sign-in confirmation feature enabled.  Old code will ignore them, new code will see not-opted-in-yet users as being verified on both counts.